src/wallet/test/util.cpp

```diff
@@ -11,10 +11,11 @@
 #include <wallet/wallet.h>
 #include <wallet/walletdb.h>
 
-#include <boost/test/unit_test.hpp>
-
 #include <memory>
 
+const std::string ADDRESS_BCRT1_UNSPENDABLE = "bcrt1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq3xueyj";
```

Comment:
In cd62eddbcfba652a238374d62f70181cb3ad9075 "refactor: unify test/util/wallet.h with wallet/test/util.h"

nit: Although this were originally in `src/test/util/wallet.h`, it doesn't really make sense to be in the wallet's utils since it is not a wallet specific address. Could probably go in `src/test/util/script.h` or `src/test/util/transaction_utils.h`